ember-server/shared/src/test/scala/org/http4s/ember/server/EmberServerSuite.scala

```diff
@@ -193,4 +196,23 @@ class EmberServerSuite extends Http4sSuite {
     }
   }
 
+  fixture().test(
+    "#7655 — client shouldn't fail request processing when server ignores the request body"
+  ) { case (server, client) =>
+    assume(
+      Platform.isJvm || Platform.isNative,
+      "Currently, the mentioned behaviour is only supported on JVM/Native",
+    )
```

Comment:
There's an ongoing discussion in discord about a discrepancy in the behaviour of `fs2.Socket` between JVM/Native and JS. There's a chance we've found an issue in fs2, and it might be resolved there, which would allow us to enable that test for the JS platform as well.